src/engine/controls/cuecontrol.cpp

```diff
@@ -628,11 +625,8 @@ void CueControl::quantizeChanged(double v) {
 }
 
 void CueControl::hotcueSet(HotcueControl* pControl, double v) {
-    //qDebug() << "CueControl::hotcueSet" << v;
-
```

Comment:
I think these are there intentionally to make it easier to debug if something goes wrong. I hope we can get rid of that when we actually make use of Qt's logging facilities.

Reply:
Yeah maybe they were, but such simple logs can easily be written again. Furthermore, I am pretty sure such control events are already logged in a way.